src/pyclvm/vscode/start.py

```diff
@@ -1,8 +1,10 @@
-import os
 import subprocess
+from time import sleep
 from typing import Any, List
 
+import psutil
```

Comment:
The `psutil` library is not in the standard library. It needs to be added as a dependency to `pyproject.toml`.

Reply:
Sure.